ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/EeaTransaction.java

```diff
@@ -32,18 +33,22 @@
   private final EeaSendTransactionJsonParameters eeaSendTransactionJsonParameters;
   private final RawPrivateTransactionBuilder rawPrivateTransactionBuilder;
   private final JsonRpcRequestId id;
+  private final NonceProvider nonceProvider;
```

Comment:
should ID and nonceProvider be moved into an abstract base?

Reply:
don't there is much benefit to do so, maybe only two lines of code reuse